src/geanyobject.h

```diff
@@ -47,6 +47,7 @@ typedef enum
 	GCB_PROJECT_OPEN,
 	GCB_PROJECT_SAVE,
 	GCB_PROJECT_CLOSE,
+	GDB_PROJECT_BEFORE_CLOSE,
```

Comment:
@b4n, @kugel- is it safe to change this enum in the middle, although its not officially in the API is it used by some object macro or something that makes it in the ABI?